api/serializers/personal_info.py

```diff
@@ -53,6 +55,36 @@ class Meta:
         extra_kwargs = {"registration_telegram_bot_chat_id": {"required": True}}
 
 
+class GetChatwootConversationIdSerializer(serializers.ModelSerializer[PersonalInfo]):
+    """A serializer used to look up Chatwoot conversation ID by Telegram bot chat ID."""
+
+    chatwoot_conversation_id = serializers.SerializerMethodField(read_only=True)
```

Comment:
MethodField is always read-only, by design

Reply:
Well, I made it doubly sure, didn't I? 😆

Reply:
This is not relevant as it got removed from the code